### PR TITLE
Print confidence and support on new line

### PR DESCRIPTION
**Pull request by zurk**
Use 3 digits for confidence.
Signed-off-by:  Konstantin Slavnov <kslavnov@gmail.com>

### CODE REVIEW

File: lookout/style/format/descriptions.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def describe_rule(rule: Rule, feature_extractor: FeatureExtractor) -> str:
         for group, nodes in grouped.items()
         for node, feature_names in nodes.items()
         for feature_name, splits in feature_names.items()]
-    return "%s\n\t→ y = %s (%.2f confidence, %d support)" % (
+    return "%s\n\t→ y = %s\n\t(%.3f confidence, %d support)" % (
```
Comment (m09) [inline lookout/style/format/descriptions.py]: if we put it on a separate line I'd rather have `Confidence: %.3f, support: %d`

File: lookout/style/format/descriptions.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def describe_rule(rule: Rule, feature_extractor: FeatureExtractor) -> str:
         for group, nodes in grouped.items()
         for node, feature_names in nodes.items()
         for feature_name, splits in feature_names.items()]
-    return "%s\n\t→ y = %s (%.2f confidence, %d support)" % (
+    return "%s\n\t→ y = %s\n\tConfidence: %.3f, support: %d." % (
```
Comment (vmarkovtsev) [inline lookout/style/format/descriptions.py]: My hidden perfectionist demands Confidence and Support or confidence and support

Comment (zurk) [reply lookout/style/format/descriptions.py]: I come up with a new word: over-reviewing 😄

File: lookout/style/format/descriptions.py [hunk 3]
```diff
@@ -66,7 +66,7 @@ def describe_rule(rule: Rule, feature_extractor: FeatureExtractor) -> str:
         for group, nodes in grouped.items()
         for node, feature_names in nodes.items()
         for feature_name, splits in feature_names.items()]
-    return "%s\n\t→ y = %s (%.2f confidence, %d support)" % (
+    return "%s\n\t→ y = %s\n\tConfidence: %.3f. Support: %d." % (
```
Comment (vmarkovtsev) [inline lookout/style/format/descriptions.py]: Noooo it is a dot instead of comma :-D

Comment (zurk) [reply lookout/style/format/descriptions.py]: Because otherwise there is no reason for `Support` instead of `support` :)

**Pull request merged**